src/components/dashboard/data/utils.js

```diff
@@ -0,0 +1,11 @@
+import { ASSIGNMENT_TYPES } from '../../enterprise-user-subsidy/enterprise-offers/data/constants';
+
+export default function getActiveAssignments(assignments) {
+  const filteredAssignments = assignments?.filter((assignment) => assignment?.state === ASSIGNMENT_TYPES.ALLOCATED
+      || assignment?.state === ASSIGNMENT_TYPES.CANCELLED);
+  const isActiveAssignments = () => filteredAssignments.length > 0;
```

Comment:
[clarification] Is this intended to be a function or a boolean? It looks like it could return a boolean value instead of a function? 

[nit] Related, perhaps `hasActiveAssignments`?